modules/ROOT/pages/about-flows.adoc

```diff
@@ -19,9 +19,7 @@ flow processes in some special way.
 You can connect and trigger their execution with xref:flowref-about.adoc[Flow Reference]
 components or by using the DataWeave `lookup` function within expressions and
 transformations. The function passes the current event to another flow for
-further event processing. In this way, you can think of a flow or subflow
-as a function call that accepts an event as input and then returns the
-modified event.
+further event processing. In this way, you can think of a flow as a function call that accepts an event as input and then returns the modified event.
```

Comment:
`... you can think of a flow as a function ...` -> `... you can think of a flow-ref as a function ...`